INSTALL.md

```diff
@@ -69,40 +69,29 @@ brew info elasticsearch
 ```
 
 ### MYSQL Database
-Ensure your database is running.
+Start MYSQL with the following command:
```

Comment:
technically this can go to, since this script is run from `./setup.sh` along with starting the server. but maybe we want to keep it in just for documentation purposes